python/with-django/README.md

```diff
@@ -0,0 +1,13 @@
+# Railway Django Example
+
+Initialize a Railway project
+
+`railway init`
+
+Run Django with Railway
+
+`railway run`
+
+/fin
+
+NOTE: The only thing changed in this project was altering the DATABASES dictionary inside of settings.py (Changed ENGINE from sqlite3 to postgresql)
```

Comment:
Since Django, Rails, etc generate random secrets for auth/cookies/etc, I don't want to commit these files

For Ruby, I've added a readme with a oneliner. We need to figure out what the Django equivalent is.